blogposts/announcing-sourcegraph-3.15.md

```diff
@@ -0,0 +1,313 @@
+<!--
+<p class="container">
+  <div style="padding:56.25% 0 0 0;position:relative;">
+    <iframe src="https://player.vimeo.com/video/{ID}?color=0CB6F4&amp;title=0&amp;byline=" style="position:absolute;top:0;left:0;width:100%;height:100%;" frameborder="0" webkitallowfullscreen="" mozallowfullscreen="" allowfullscreen=""></iframe>
+  </div>
+  <p style="text-align: center"><a href="https://vimeo.com/{ID}" target="_blank">View on Vimeo</a></p>
+</p>
+-->
+
+The new `visibility:` search scope enables filtering based on a repository’s shared state on the code host. Scope options include any, public, or private, and defaults to `visibility:any` when the scope is not specified.
+
+## Setting to include forked and archived repositories
+
+Sourcegraph 3.14 removed archived and forked repositories from search results by default. While this was an improvement to most users, some organizations have a configuration that necessitates searching these repositories. Sourcegraph 3.15 enables those organizations to add a global setting to their instance to always include forks or archived repositories in searches. Enable this in global, organization, or user settings with `"search.includeForks": true` or `"search.includeArchived": true` respectively.
+
+## Observability
+
+We continue to invest heavily in making Sourcegraph easier to monitor and debug out-of-the-box.
+
+### Monitoring
+
+- **Sourcegraph now monitors and has pre-defined alerts for 93 metrics across 14 services.** For now, these new alerts are warning-level alerts and in the future once more confidence in their accuracy has been gained they will be elevated to critical-level alerts.
+- 8 bug fixes to alerting and dashboards have been made
+- 11 changes to improve the legibility and visibility of dashboards have been made
+
+Additionally:
+
+- All dashboard panels now show an orange/red background when a metric is reaching its warning/critical threshold, making it easy to see which metrics are most interesting.
+- Search error rate and latency is now broken down by type on the Frontend dashboard:
+  - Search at a glance (browser web UI)
+  - Search-based code intelligence at a glance
+  - Search API usage at a glance
+
+**IMPORTANT:** If you have previously configured alert thresholds on the Frontend dashboard manually (instead of just on the Grafana Home dashboard as is typically reccomended), you will need to reconfigure those alert thresholds again after upgrading.
+
+![image](https://user-images.githubusercontent.com/3173176/79792360-9e2b4d80-8303-11ea-91e5-cb7e9c972358.png)
+
+### Debugging
+
+We've added a new tool to all Sourcegraph deployments by default: Distributed tracing. It is a powerful tool for investigating performance issues. You can enable tracing selectively per-request and capture detailed information about, for example, a search request which can then easily be inspected to find out what the slowest parts were within the inner-workings of Sourcegraph.
+
+TODO: screenshot
```

Comment:
@beyang can you provide a screenshot of Jaeger on a search request?

Reply:
I went ahead and merged without it, but would love to get this added in later :)